tests/test_connector.rs

```diff
@@ -457,12 +458,23 @@ fn test_deposit_with_same_proof() {
         p.assert_success()
     }
     let promises = call_deposit_near(&contract, CONTRACT_ACC);
-    let l = promises.len();
-    let p = promises[l - 2].clone();
-    match p.unwrap().status() {
-        ExecutionStatus::Failure(_) => {}
-        _ => panic!(),
-    }
+    let promise = &promises[promises.len() - 2];
+    check_execution_status_failure(
+        promise.as_ref().unwrap().outcome().clone().status,
+        "ERR_PROOF_EXIST",
+    );
+}
+
+#[test]
+fn test_deposit_wrong_custodian_address() {
+    let custodian_address = "096DE9C2B8A5B8c22cEe3289B101f6960d68E510";
```

Comment:
Do we have any reason to set this specific address to ensure that the deposit fails if the proof is coming from `EthCustodian` that was not specified for this contract?
I'd prefer to set the initial value for the address as `0000000000000000000000000000000000000001`, so it's clear that we are not checking for some specific value.

Reply:
It's just wrong address. I think we don't need zero account.

Reply:
The purpose of this test is to check the mismatch for `EthCustodian` addresses. So it's not very clear from where the specified value is coming from. But for sure this should be wrong address, so I think it's ok if we specify the hardcoded account with `1` address (not zero)